### ISSUE TITLE
Custom icons on folder not well supported

### ISSUE THREAD

**Issue by Xample** [LinusU/node-appdmg#50]
I do have several folders with a custom icon i.e. the ```bash Icon\r``` file within the folder itself and the folder has the ```bash SetFile -a C``` property (flag for specifying a custom icon).
However it seems that while including those folders into the .DMG, the custom icon is present on my computer but disappears on another one.

Possible resolution: I would suggest to add an official support for the content custom icons i.e. the ability of specifying an "icon" property to the content type the same way we can do it for the disk image.

**Comment by LinusU** [comment]
Hmm, I'm not sure I understand exactly what is going on. Do you have a folder with a custom icon that you add to the dmg, and that icon then disappears when opening the dmg on another computer?

It might be related to what I said in #36 and I would like to say the same thing here.

> If that dosen't work (e.g. cp -r dosen't copy extended attributes) than we can reopen this as Extended attributes is not preserved.

**Comment by Xample** [comment]
To be accurate:
1. I do have a folder with a custom icon
2. I do copy the icon ```/Icon\r``` of this folder to another folder and active it with ```SetFile -a C```
At this stage I can see the icon on both folders
3. I add the second folder as a dependency into node-appdmg and build
4. The disk image is built, the icon of the folder appears correctly on my computer
5. I sent the dmg to a second computer the icon do not appears this time.

Perhaps using ```ditto```bash instead of ```cp -r``` would fix the issue, however as I saw, the icon was copied into the archive. Also it does not explains why it is working on a computer and not on another one. My first computer is retina, the second one is not.

And still: if I could specify an ```icon``` property within the .json description file it would just be awesome. I'm doing my own kitchen with the icon file copy because I did not found any tool doing it properly yet.
